CHANGELOG.md

```diff
@@ -2,11 +2,8 @@
 
 ## unreleased
 
-- **(feat)** Add new two plot types: point and bar. You can change the plot type by selecting the graph, and modifying the plot type in the inspector panel.
```

Comment:
Prolly don't want to delete these change logs.

Reply:
Fixed